extensions-contrib/kafka-emitter/src/main/java/org/apache/druid/emitter/kafka/KafkaEmitterConfig.java

```diff
@@ -259,6 +270,9 @@ public boolean equals(Object o)
     if (!getKafkaProducerConfig().equals(that.getKafkaProducerConfig())) {
       return false;
     }
+    if (!getShutdownTimeout().equals(that.getShutdownTimeout())) {
```

Comment:
possible NPE since shutdownTimeout is nullable?

Reply:
It won't cause an NPE because the default `Long.MAX_VALUE` value is assigned to the member variable in the ctr, but yeah the nullable annotation needs to be removed